controllers/article.controller.js

```diff
@@ -1,19 +1,31 @@
+const articles = require("../db/data/test-data/articles")
 const {fetchArticles} = require("../models/articles.model")
 exports.getArticle = (req,res, next) => {
     const article_id = req.params.article_id
     return fetchArticles()
     .then((articles) => {
-        console.log(articles)
         const article = articles.find((article) => {
             return article.article_id == article_id
```

Comment:
This isn't very efficient - you have a lot of data requested that you don't want to use, so it's much better to write a separate query in your models that is more specific and only gets the article you want.

Reply:
I can see that Jodie has given you feedback on that that definitely would've required a resubmission, we have not received that yet from you. I expect to see resubmissions to these today.